### ISSUE TITLE
Error when building with Docusaurus 2.3.0

### ISSUE THREAD

**Issue by TheCatLady** [PaloAltoNetworks/docusaurus-openapi-docs#396]
## Describe the bug

Tabs were refactored in facebook/docusaurus#8486

## Expected behavior

No error when building with Docusaurus 2.3.0.

## Current behavior

Error when building:
```
TypeError: useTabGroupChoice is not a function
```

## Possible solution

Fix usage of `useTabGroupChoice`.

## Steps to reproduce

1. Update Docusaurus package versions in `package.json`
2. `yarn`
3. `yarn build`

## Context

I am unable to update Docusaurus to the new 2.3.0 release.

**Comment by dio** [comment]
I also want to report the problem with docusaurus 2.3.0. It can be reproduced by cloning https://github.com/dio/docusaurus-template-openapi-docs/tree/2.3.0 and do `yarn`, then `yarn build`.

```
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/tutorial-extras/translate-your-site.


ReactContextError
```

Complete logs:

```
$ yarn build
yarn run v1.22.19
$ docusaurus build
[INFO] [en] Creating an optimized production build...

✔ Client
  

● Server █████████████████████████ sealing (88%)  
 runtime requirements

[ERROR] Docusaurus server-side rendering could not render static page with path /docs/intro.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/add-pet.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/create-user.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/create-users-with-array-input.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/create-users-with-list-input.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/delete-order.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/delete-pet.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/delete-user.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/find-pets-by-status.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/find-pets-by-tags.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/get-inventory.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/get-order-by-id.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/get-pet-by-id.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/get-user-by-name.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/login-user.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/logout-user.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/pet.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/place-order.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/store.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/subscribe-to-the-store-events.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/swagger-petstore-yaml.

✔ Client
  

● Server █████████████████████████ sealing (92%) asset processing 
 RealContentHashPlugin

[ERROR] Docusaurus server-side rendering could not render static page with path /docs/intro.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/add-pet.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/create-user.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/create-users-with-array-input.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/create-users-with-list-input.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/delete-order.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/delete-pet.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/delete-user.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/find-pets-by-status.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/find-pets-by-tags.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/get-inventory.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/get-order-by-id.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/get-pet-by-id.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/get-user-by-name.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/login-user.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/logout-user.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/pet.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/place-order.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/store.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/subscribe-to-the-store-events.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/swagger-petstore-yaml.

✔ Client
  

✖ Server
  Compiled with some errors in 20.47s

[ERROR] Docusaurus server-side rendering could not render static page with path /docs/intro.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/add-pet.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/create-user.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/create-users-with-array-input.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/create-users-with-list-input.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/delete-order.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/delete-pet.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/delete-user.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/find-pets-by-status.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/find-pets-by-tags.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/get-inventory.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/get-order-by-id.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/get-pet-by-id.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/get-user-by-name.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/login-user.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/logout-user.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/pet.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/place-order.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/store.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/subscribe-to-the-store-events.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/swagger-petstore-yaml.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/update-pet.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/update-pet-with-form.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/update-user.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/upload-file.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/petstore/user.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/tutorial-basics/congratulations.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/tutorial-basics/create-a-blog-post.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/tutorial-basics/create-a-document.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/tutorial-basics/create-a-page.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/tutorial-basics/deploy-your-site.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/tutorial-basics/markdown-features.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/tutorial-extras/manage-docs-versions.
[ERROR] Docusaurus server-side rendering could not render static page with path /docs/tutorial-extras/translate-your-site.


ReactContextError
ReactContextError
ReactContextError
ReactContextError
ReactContextError
ReactContextError
ReactContextError
ReactContextError
ReactContextError
ReactContextError
ReactContextError
ReactContextError
ReactContextError
ReactContextError
ReactContextError
ReactContextError
ReactContextError
ReactContextError
ReactContextError
ReactContextError
ReactContextError
ReactContextError
ReactContextError
ReactContextError
ReactContextError
ReactContextError
ReactContextError
ReactContextError
ReactContextError
ReactContextError
ReactContextError
ReactContextError
ReactContextError
ReactContextError
[ERROR] Unable to build website for locale en.
[ERROR] Error: Failed to compile with errors.
    at /Users/dio/src/dio/docusaurus-template-openapi-docs/node_modules/@docusaurus/core/lib/webpack/utils.js:180:24
    at /Users/dio/src/dio/docusaurus-template-openapi-docs/node_modules/webpack/lib/MultiCompiler.js:554:14
    at processQueueWorker (/Users/dio/src/dio/docusaurus-template-openapi-docs/node_modules/webpack/lib/MultiCompiler.js:491:6)
    at processTicksAndRejections (node:internal/process/task_queues:78:11)
[INFO] Docusaurus version: 2.3.0
Node version: v16.19.0
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by zerosoul** [comment]
When openapi theme plugin sync the refactored tab component?

**Comment by benlei** [comment]
Is the current recommendation to just not use 2.3 (i.e. use 2.2)?

**Comment by sserrata** [comment]
Hi @benlei, yes, that's correct. The next release (1.5.3) will pin Docusaurus to `>=2.0.1 <2.3.0` as we work on v2.0.0.

**Comment by sserrata** [comment]
Support for 2.3.0 available in v2.0.0-beta.1

**Comment by gastonfournier** [comment]
> Support for 2.3.0 available in v2.0.0-beta.1

Thanks!! Is there any expected timeline for 2.0.0? We're considering whether we should use the beta.1 version or wait for the stable release.
